bin/deployment-rds.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -ex
+
+export AWS_ACCESS_KEY_ID=$bamboo_AWS_ACCESS_KEY_ID
+export AWS_SECRET_ACCESS_KEY=$bamboo_AWS_SECRET_ACCESS_KEY
+export AWS_DEFAULT_REGION=$bamboo_AWS_DEFAULT_REGION
+
+#clone cumulus orca template for deploying cumulus and orca
+git clone https://git.earthdata.nasa.gov/scm/orca/cumulus-orca-deploy-template.git
+cd cumulus-orca-deploy-template
+git checkout $bamboo_ORCA_RELEASE_BRANCH
+echo "checked out to $bamboo_ORCA_RELEASE_BRANCH branch"
+
+
+#deploy rds-cluster-tf module
+
+cd rds-cluster-tf
+echo "inside rds-cluster-tf"
+mv terraform.tfvars.example terraform.tfvars
+
+RDS_CLUSTER_KEY="$bamboo_PREFIX/rds-cluster-tf/terraform.tfstate"
+# Ensure remote state is configured for the deployment
+echo "terraform {
```

Comment:
See previous notes on templating the files

Reply:
I followed the styling of what cumulus uses
See https://github.com/nasa/cumulus/blob/87b78010ffc85862585c9b123829573b2151385d/bamboo/bootstrap-tf-deployment.sh#L61

Reply:
The current rds-cluster-tf/terraform.tf.example has this format https://git.earthdata.nasa.gov/projects/ORCA/repos/cumulus-orca-deploy-template/browse/rds-cluster-tf/terraform.tf.example?at=release%2Fv11.1.1-v4.0.1

    terraform {
      backend "s3" {
        region         = "<region>"
        bucket         = "<state-bucket>"
        key            = "<state-path>/terraform.tfstate"
        dynamodb_table = "<lock-table-name>"
      }
    }
    
    so we should update this file with the correct format before using sed command like in other modules.
    
    Using echo for now in this case.

Reply:
already changed in the template